storage.bs

```diff
@@ -234,7 +235,16 @@ interface StorageManager {
 dictionary StorageEstimate {
   unsigned long long usage;
   unsigned long long quota;
+  StorageUsageDetails usageDetails;
 };
+
+dictionary StorageUsageDetails {
+    unsigned long long applicationCache;
```

Comment:
Style: indentation is inconsistent with previous lines (appears to be two spaces)